aerogear/mobile-docs
# Updated service specific info on binding

## Motivation
https://issues.jboss.org/browse/AEROGEAR-8225
https://issues.jboss.org/browse/AEROGEAR-8235

## What
Updated service specific info on binding in MDC.

## Why
There was stilll invalid docs about binding/undbinding specific services.

## Verification Steps
Add the steps required to check this change. Following an example.
 
1. Go to `some service > Setting up` section
2. There should be valid docs

## Checklist:
